endpoint.go

```diff
@@ -127,23 +113,33 @@ func (e *Endpoint) Addr() net.Addr {
 	return e.addr
 }
 
-func (e *Endpoint) WriteReliablePacket(buf []byte, addr net.Addr) error {
+func (e *Endpoint) WriteReliablePacket(buf []byte, addr net.Addr, opts ...ConnOption) error {
```

Comment:
warning: exported method Endpoint.WriteReliablePacket should have comment or be unexported (golint)